test/functional/wallet-hd.py

```diff
@@ -15,17 +15,11 @@ class WalletHDTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
-        self.extra_args = [['-usehd=0'], ['-usehd=1', '-keypool=0']]
+        self.extra_args = [[], ['-keypool=0']]
 
     def run_test (self):
         tmpdir = self.options.tmpdir
 
-        # Make sure can't switch off usehd after wallet creation
-        self.stop_node(1)
-        self.assert_start_raises_init_error(1, ['-usehd=0'], 'already existing HD wallet')
```

Comment:
Why is this test removed?

Reply:
Oops. That was removed originally when I had `-usehd` always return an error regardless.